css/home-page.styl

```diff
@@ -110,6 +112,10 @@ $flex-section
         padding: 0 10vw
         text-align: left
 
+      @media (max-width: 768px)
+        max-width: 100%
+        padding: 0 2em
+
```

Comment:
Text was too skinny: 

<img width="288" alt="screen shot 2019-01-23 at 10 17 05 pm" src="https://user-images.githubusercontent.com/3202211/51653839-40bf1a00-1f63-11e9-832b-aedeb9b9ad74.png">

The fix: 
<img width="285" alt="screen shot 2019-01-23 at 10 18 21 pm" src="https://user-images.githubusercontent.com/3202211/51653851-4d437280-1f63-11e9-8177-5fa09bc7d5dd.png">